packages/web-api-scan-runner/src/crawl-runner/crawl-runner.ts

```diff
@@ -19,14 +18,12 @@ export class CrawlRunner {
         @inject(crawlerIocTypes.CrawlerProvider) private readonly getCrawler: CrawlerProvider,
         @inject(GlobalLogger) private readonly logger: GlobalLogger,
         @inject(ServiceConfiguration) private readonly serviceConfig: ServiceConfiguration,
-        @inject(ScanMetadataConfig) scanMetadataConfig: ScanMetadataConfig,
         @inject(BatchConfig) private readonly batchConfig: BatchConfig,
-    ) {
-        const scanMetadata = scanMetadataConfig.getConfig();
-        this.logger.setCommonProperties({ scanId: scanMetadata.id });
```

Comment:
Moved this into run() because it was causing our dependency injection tests to fail (we expect logger.setup() to be called before any other logger methods, so calling logger methods in the constructor of an auto-injected object makes it more complicated to guarantee that)